packages/fxa-auth-server/config/dev.json

```diff
@@ -36,6 +36,9 @@
     "maxTTL": "28 days"
   },
   "subhub": {
+    "enabled": true,
+    "url": "http://127.0.0.1:8012/",
+    "key": "abcde",
     "useStubs": true,
```

Comment:
TL;DR: to use a local subhub instance here...

1. Switch `useStubs` to `false`
1. Get a Stripe API key pair
1. Run `fxa-payments-server` with env var `STRIPE_API_KEY` equal to your Stripe publishable key
1. Run this docker container with your Stripe secret key: 
    `docker run -ti --rm -p 8012:8012 -e STRIPE_API_KEY=sk_test_TESTING lmorchard/subhub:dev`